chef/cookbooks/rabbitmq/recipes/rabbit.rb

```diff
@@ -81,7 +81,10 @@
   only_if only_if_command if ha_enabled
 end
 
-if node[:rabbitmq][:trove][:enabled]
```

Comment:
Yeah, I'm interesting too on what purpose it was done.

Right now, only one possible way to enable trove user in rabbitmq is to create this attribute manually.